src/demo/App.tsx

```diff
@@ -0,0 +1,208 @@
+import * as React from 'react';
+import Demo from './Demo'
+import Tour from '../index'
+import Text from './Text'
+import Glitch from './Glitch'
+import Tooltip from './Tooltip'
+import { Link } from './Button'
+import PropTypes from 'prop-types'
+
+import './styles.css'
+
+interface IState{
+	isTourOpen: boolean,
+	isShowingMore: boolean
+}
+interface IDemoHelperComponentProps{
+	goTo: Function,
+}
+class App extends React.Component<{}, IState> {
+  constructor() {
+    super( {} )
+    this.state= {
+      isTourOpen: false,
+      isShowingMore: false,
+    }
+  }
+
+  componentDidMount() {
+    window.addEventListener( 'keyup', this.keyHandling )
+  }
+
+  componentWillUnmount() {
+    window.removeEventListener( 'keyup', this.keyHandling )
+  }
+
+  keyHandling = ( e: KeyboardEvent ) => {
+    if (e.keyCode === 75) {
+      e.preventDefault()
+      this.openTour()
+    }
+  }
+
+  toggleShowMore = () => {
+    this.setState( prevState => ( {
```

Comment:
I think you can use your _IState_ interface to be sure that prevState has all attrs you want, but i dont think it necesary cause it can be assumed by context.

Reply:
I could've but didn't. :-)